src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Figure implements DrawFigure, AreaCalculator {
```

Comment:
`Figure` should be an abstract class:
1. We shouldn't allow to create a `Figure` instance - we don't know which figure it is.
2. If you use an abstract class, you can delegate the implementation of interface methods to subclasses.